core-services/prow/02_config/_labels.yaml

```diff
@@ -651,6 +651,23 @@ repos:
         target: both
         prowPlugin: label
         addedBy: label
+  openshift/console-operator:
+    labels:
+      - color: 0FFA16
+        description: Signifies that QE has signed off on this PR
+        name: qe-approved
+        target: both
+        addedBy: label
+      - color: 0FFA16
+        description: Signifies that Product Support has signed off on this PR
+        name: px-approved
+        target: both
+        addedBy: label
+      - color: 0FFA16
+        description: Signifies that Docs has signed off on this PR
+        name: docs-approved
+        target: both
+        addedBy: label
```

Comment:
/hold

This shojld not be necessary, they are defined on org-level

Reply:
Thank you, this changed since I originally opened the PR. I'll close.